scripts/rollup/forks.js

```diff
@@ -114,12 +114,57 @@ const forks = Object.freeze({
     return null;
   },
 
-  'shared/ReactScheduler': (bundleType, entry) => {
+  'react-scheduler': (bundleType, entry, dependencies) => {
     switch (bundleType) {
       case FB_WWW_DEV:
       case FB_WWW_PROD:
       case FB_WWW_PROFILING:
         return 'shared/forks/ReactScheduler.www.js';
+      case UMD_DEV:
+      case UMD_PROD:
+        if (dependencies.indexOf('react') === -1) {
+          // For CJS bundles, use the shared NPM package.
```

Comment:
This comment doesn’t seem to make sense inside UMD conditionn

Reply:
Hah, you're right.

Reply:
I meant for that comment to be in the below `default` case. I just rebased and cleaned that one up.